### PR TITLE
Added store copy examples and error information

### PR DESCRIPTION
**Pull request by 55octet**
**Please check below, if the PR fulfills these requirements:**
- [x] Commit(s) and code follow the repositories guidelines.
- [ ] Test(s) have been added or updated to support these change(s).
- [ ] Doc(s) have been added or updated to support these change(s).

**Associated Links:**

- 

**Types of Changes:**

- Add validation and examples around the store copy command as it is did not list registry:// or dir:// for the remote repo.

**Proposed Changes:**

- Add validation and examples around the `store copy` command

**Verification/Testing of Changes:**

- Run `hauler store copy` to display examples.

**Additional Context:**

- This should be a small change to make the cli experience usable without looking at the source code.

### CODE REVIEW

**Comment by 55octet** [comment]
Based from https://github.com/rancherfederal/hauler/pull/238.

Signed my commit (hadn't updated my pgp on github) and ran `go fmt`.  I didn't notice any changes after formatting; it seems that `cmd/hauler/cli/store.go` wasn't formatted correctly.  If I am missing a format configuration, I can fix it.

**Comment by zackbradys** [comment]
hey @55octet, thank you for your contribution to the project. taking a look at the examples in the subcommand, could you please update them to `registry://registry.example.com` and `dir://hauler-files`?

**Comment by zackbradys** [comment]
@amartin120 please take a look at the error information and updated formatting.

**Comment by 55octet** [comment]
Hi @zackbradys --

I will update and push today.

Thank you!!

**Comment by zackbradys** [comment]
Hey @55octet, please update this PR when you get a chance so we can merge it!

File: cmd/hauler/cli/store/copy.go
Comment (dweomer) [inline cmd/hauler/cli/store/copy.go:1]: These don't amount to meaningful changes and I would argue make the logic less clear in that it introduces a separate `if` conditional that was already handled in the still-relevant `switch` block.

**Review by dweomer — CHANGES_REQUESTED** [review]
The example text looks great. The other changes are unnecessary.

File: cmd/hauler/cli/store.go [hunk 1]
```diff
@@ -1,9 +1,9 @@
 package cli
 
 import (
+	"fmt"
 	"github.com/spf13/cobra"
 	"helm.sh/helm/v3/pkg/action"
-	"fmt"
 
 	"github.com/rancherfederal/hauler/cmd/hauler/cli/store"
 )
```
Comment (dweomer) [inline cmd/hauler/cli/store.go:9]: can we get a `goimports --local -w` pass?

**Pull request closed without merging**